src/checker.rs

```diff
@@ -445,7 +465,10 @@ mod test {
         let end = start.elapsed();
 
         assert!(matches!(res, Status::Failed(_)));
-        assert!(matches!(end.as_secs(), 7));
+
+        // on slow connections, this might take a bit longer than nominal backed-off timeout (7 secs)
+        assert!(end.as_secs() >= 7);
+        assert!(end.as_secs() <= 8);
```

Comment:
Think there's a way to specify an epsilon. 🤔

Reply:
Nope. There are [some workarounds](https://stackoverflow.com/questions/30856285/assert-eq-with-floating-point-numbers-and-delta), but nothing built-in. 😬